hack/make/.go-autogen

```diff
@@ -2,12 +2,14 @@
 
 rm -rf autogen
 
-mkdir -p autogen/dockerversion
-cat > autogen/dockerversion/dockerversion.go <<DVEOF
-// AUTOGENERATED FILE; see $BASH_SOURCE
-package dockerversion
-
-var (
+cat > version/version_autogen.go <<DVEOF
```

Comment:
Since this file isn't in that `autogen` directory anymore, do you think we should add it to `.gitignore` too?